test/integration/noderesourcetopology_test.go

```diff
@@ -305,7 +305,7 @@ func TestTopologyMatchPlugin(t *testing.T) {
 		{
 			name: "Scheduling of a burstable pod requesting only memory",
 			pods: []*v1.Pod{
-				withContainer(st.MakePod().Namespace(ns).Name("topology-aware-scheduler-pod").Req(map[v1.ResourceName]string{v1.ResourceMemory: "5Gi"}).Obj(), pause),
+				st.MakePod().Namespace(ns).Name("topology-aware-scheduler-pod").Req(map[v1.ResourceName]string{v1.ResourceMemory: "5Gi"}).Obj(),
```

Comment:
why is `withContainer` no longer needed?

Reply:
```withContainer``` is misleading because ```Req``` is already creating a container with given resources.
It is also redundant because ```withContainer```  was setting the image and container name, but it does that with the exact values that ```Req``` already provided.

Reply:
thanks, this fully make sense.